### PR TITLE
Update interfaces for the hall events module

### PR DESCRIPTION
**Pull request by jieffyy**
As part of #92, #95.

Update the `Model.java` and `Logic.java` interfaces to handle the new hall events module. \
Also left a `TODO: [Event]` in the code so I know what to update for the next PR.

Changes are mainly to provide support for the hall event UI side for @yanchenglee98. \
Also added one functioning test case to show how to extract the list.

### CODE REVIEW

File: src/main/java/seedu/address/model/event/Event.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.event;
+
+public class Event {
+    //TODO: [Event] Complete the class
+    private final String eventName;
+
```
Comment (yanchenglee98) [inline src/main/java/seedu/address/model/event/Event.java]: Each event will still need to have a list of students i.e. List<Person> to keep track which students are attending which event.
Will need an accessor to the list and a way to add students to the list also.

**Review by yanchenglee98 — COMMENTED** [review]
Some details of event class still need to be iron out.

File: src/main/java/seedu/address/model/event/Event.java (hunk 1 above)
Comment (jieffyy) [reply src/main/java/seedu/address/model/event/Event.java]: Okay, I have added a field to manage the list of attendees tot the `Event` class.

I didn't add the way to add students to the list yet - this is still a dummy implementation and there might be some design considerations you can make for associating persons to events (e.g. unmodifiable lists, etc).

**Review by yanchenglee98 — APPROVED** [review]
Hall event interface looks workable. Merging

**Comment by jieffyy** [comment]
EDIT: was part of #91 as well.

**Pull request merged**